config/environment.js

```diff
@@ -144,6 +144,7 @@ module.exports = function(environment) {
             },
         },
         OSF: {
+            providerId: 'osf',
```

Comment:
let's make this more descriptive -- maybe `osfRegistrationProviderId`? there are other types of providers, unfortunately